app/src/main/AndroidManifest.xml

```diff
@@ -37,7 +37,9 @@
         android:name=".FocusApplication"
         android:usesCleartextTraffic="true">
 
-        <activity android:name=".activity.IntentReceiverActivity">
+        <activity
+            android:name=".activity.IntentReceiverActivity"
+            android:exported="true">
```

Comment:
Why is this exported?
Seems like an important change that should be better documented - in the ticket and in commit message.

Reply:
To keep the action  <action android:name="android.intent.action.VIEW" />